seed/static/seed/js/controllers/data_quality_admin_controller.js

```diff
@@ -153,23 +153,12 @@ angular.module('BE.seed.controller.data_quality_admin', [])
       $scope.isModified = function () {
         return modified_service.isModified();
       };
-      var originalRules = angular.copy(data_quality_rules_payload.rules);
-      $scope.original = originalRules;
-      $scope.change_rules = function () {
+
+      $scope.rules_changed = function () {
         $scope.defaults_restored = false;
         $scope.rules_reset = false;
         $scope.rules_updated = false;
-        $scope.setModified();
-      };
-      $scope.setModified = function () {
```

Comment:
Effectively, this only ever triggered the `modified_service.setModified()` method, so I simplified this logic.

From what I could tell, this method was attempting to check that the previous rules were still equal. I'm assuming that the intent was to eventually "re-disable" the "Save Changes" button if all rules were unchanged. But, I don't think this quite got there. 

Notably, there were some challenges in checking if rules have changed, including the potential for the inclusion of a $$hashkey to the changed rule as well as the potential to have a nested object (in the form of a label) for each rule object. 

I think it is possible to do, but it isn't as straight forward, so I took the easy path for now. Let me know if I should spend time on this.